client/my-sites/plans/jetpack-plans/selector.tsx

```diff
@@ -221,7 +224,19 @@ const SelectorPage: React.FC< SelectorPageProps > = ( {
 				/>
 
 				{ isEnabled( 'jetpack/pricing-page-rework-v1' ) ? (
-					<ProductStore enableUserLicensesDialog={ enableUserLicensesDialog } />
+					<>
+						{
+							<PricingHeader
```

Comment:
Hmm, I'm not an expert but I think we try using **context** to specify things like Header, Footer etc. I'd be curious to read some documentation on this matter but haven't found one. 
In the meantime I suggest try to follow current implementation and rather specify title in `client/jetpack-cloud/sections/pricing/controller.tsx` based on Feature Flag.

Reply:
Thanks for pointing this out Andrii. This indeed is the right place for declaring the header.

I have move the declaration within the controller context as suggested. 👍